CHANGELOG.md

```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.9.0] - 2019-01-18
+### Breaking
+- The `lib/linter.js` module has been entirely removed, and oas-linter is now used directly ([#248])
+- Rulesets must now be YAML not JSON ([#248])
```

Comment:
If we're following semantic versioning, for breaking changes shouldn't we increment the major version?

Reply:
@smythey21 #WellActually development versions can contain breaking changes on a minor.

Reply:
Indeed! Good to know 👍